galleries/examples/color/individual_colors_from_cmap.py

```diff
@@ -7,39 +7,39 @@
 cycle provides. Selecting individual colors from one of the provided colormaps can be a
 convenient way to do this.
 
-Once we have hold of a `.Colormap` instance, the individual colors can be accessed
-by passing it an index.  If we want a specific number of colors taken at regular
-intervals from a continuous colormap, we can create a new colormap using the
-`~.Colormap.resampled` method.
-
-For more details about manipulating colormaps, see :ref:`colormap-manipulation`.
+Once we have hold of a `.Colormap` instance, individual colors can be accessed by
+passing one or more floats or integers.  For more details about manipulating colormaps,
+see :ref:`colormap-manipulation`.
 """
 
 import matplotlib.pyplot as plt
+import numpy as np
 
 import matplotlib as mpl
 
 n_lines = 21
+cmap = mpl.colormaps['plasma']
 
-cmap = mpl.colormaps.get_cmap('plasma').resampled(n_lines)
+# Take colors at regular intervals spanning the color map.
+colors = cmap(np.linspace(0, 1, n_lines))
 
 fig, ax = plt.subplots(layout='constrained')
 
-for i in range(n_lines):
-    ax.plot([0, i], color=cmap(i))
+for i, color in enumerate(colors):
+    ax.plot([0, i], color=color)
 
 plt.show()
 
 # %%
 # Instead of passing colors one by one to `~.Axes.plot`, we can replace the default
 # color cycle with a different set of colors.  Specifying a `~cycler.cycler` instance
-# within `.rcParams` achieves that.  See :ref:`color_cycle` for details.
+# within `.rcParams` achieves that (see :ref:`color_cycle` for details).  The list of
+# all colors in a `.ListedColormap` is available as the ``colors`` attribute.
 
 
 from cycler import cycler
 
-cmap = mpl.colormaps.get_cmap('Dark2')
-colors = cmap(range(cmap.N))  # cmap.N is number of unique colors in the colormap
+colors = mpl.colormaps['Dark2'].colors
```

Comment:
#27678 started by using the `colors` attribute but  @jklymak said it could be confusing as `LinearSegmentedColormap` doesn't have it.  Then @timhoffm suggested using it.  I am hoping that using after specifically stating that it's on `ListedColormap`s is a reasonable compromise.

Reply:
Sectioning: The above should be something "Extracting values from a continuous colormap" (though technically most of the visually continous colormaps are still densely sampled ListedColormaps) and here "Extracting values from a discrete colormap".

I also suggest to not combine both the `ListedColormap.colors` explanation and the the `rc_context` into one plot. That's confusing. Either drop the `rc_context`, or make it a separate plot.